src/main/java/com/amihaiemil/docker/UnexpectedResponseException.java

```diff
@@ -50,20 +52,29 @@
      */
     private final int expectedStatus;
 
+    /**
+     * The response's body.
+     */
+    private final JsonObject payload;
+
     /**
      * Ctor.
      * @param endpoint Endpoint that was called.
      * @param actualStatus Received status.
      * @param expectedStatus Expected status.
+     * @param body The response's body.
      */
+    // @checkstyle ParameterNumber (3 lines)
     public UnexpectedResponseException(
-        final String endpoint, final int actualStatus, final int expectedStatus
+        final String endpoint, final int actualStatus,
+        final int expectedStatus, final JsonObject body
     ) {
         // @checkstyle LineLength (1 line)
         super("Expected status " + expectedStatus + " but got " + actualStatus + " when calling " + endpoint);
```

Comment:
@llorllale Can we also add the body in the message above (when calling ``super("Expected...")``?

Reply:
@amihaiemil ok